flow/layers/clip_rect_layer.h

```diff
@@ -16,10 +16,7 @@ class ClipRectLayer : public ContainerLayer {
 
   void Preroll(PrerollContext* context, const SkMatrix& matrix) override;
   void Paint(PaintContext& context) const override;
-
-#if defined(OS_FUCHSIA)
   void UpdateScene(SceneUpdateContext& context) override;
```

Comment:
@chinmaygarde : what's your take on `UpdateScene` and `SceneUpdateContext` in the future?